### PR TITLE
[Test] add sync option in appsink

### PR DESCRIPTION
**Pull request by jaeyun-jung**
1. add sync=false option in model reload test.
2. update pipeline description in docs.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: tests/nnstreamer_filter_reload/tensor_filter_reload_test.c [hunk 1]
```diff
@@ -249,6 +249,7 @@ main (int argc, char *argv[])
   g_object_set (G_OBJECT (tensor_filter), "is-updatable", TRUE, NULL);
 
   g_object_set (G_OBJECT (appsink), "emit-signals", TRUE, NULL);
+  g_object_set (G_OBJECT (appsink), "sync", TRUE, NULL);
```
Comment (myungjoo) [inline tests/nnstreamer_filter_reload/tensor_filter_reload_test.c:252]: ```s/TRUE/FALSE/```?

Comment (jaeyun-jung) [reply tests/nnstreamer_filter_reload/tensor_filter_reload_test.c]: my fault, changed to 'false'.

**Comment by gichan-jang** [comment]
Q. Is there a reason why the sync option of appsink is not set to the default false?

**Comment by jaeyun-jung** [comment]
@gichan-jang 
It is not appsink, basesink has default false.

**Pull request merged**